views/managing-officer-corporate.html

```diff
@@ -0,0 +1,303 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Corporate managing officer – {{ serviceName }} – GOV.UK
```

Comment:
need to replace service name with the name of our app (see other templates) and the " – " character is flagged as invalid in the vscode editor  - The character U+2013 "–" could be confused with the character U+002d "-", I would change them to be minus signs.

Reply:
does the {{ serviceName }} get replaced? i had replaced it with text on mine but perhaps I didn't need to, does it get replaced in your docker env?